packages/db/test/user.spec.js

```diff
@@ -66,6 +66,7 @@ describe('user operations', () => {
     const name = 'test-name'
     const email = 'new-test@email.com'
     const publicAddress = `public_address${Math.random()}`
+    const pinningEnabled = true
```

Comment:
This isn't used. 
Can we get rid of it?

Reply:
Yes, removed.